doc/OutlierFiltersFamilies.md

```diff
@@ -0,0 +1,377 @@
+
+![linear Family plot](images/linear_family.png)
+
+#### L2 <a name="l2head"></a>
+
+L2 is a function which is a part of the Lp space family of functions. The L2 outlier filter can't be tuned with a k parameter.
+
+Equations:
+![eq l2](images/eq_l2.JPG)
+
+Plot:
+![L2 nok](images/L2_nok.png)
+
+#### Switchable-Constraint <a name="switchableconstrainthead"></a>
+
+Equations:
+![eq sc](images/eq_sc.JPG)
+
+Plots:
+![switchableconstraint threek](images/switchableconstraint_threek.png)
+![switchableconstraint multik](images/switchableconstraint_multik.png)
+
+Original reference:
+N. Süderhauf and P. Protzel,"Switchable constraint for robust pose graph slam", in 2012 IEEE/RSJ International Conference on Intelligent Robots and Systems, 2012, pp. 1879-1884. & P Argawal, G.D. Tipaldi, L. Spinello, C.Stachniss, and W.Burgard, "Robust map optimization using dynamic covariance scaling", IEEE international Conference on Robotics and Automation, pp. 62-69, 2013.
+
+#### Maximum Distance (MaxDist) <a name="maximumdistancehead"></a>
+
+The Maximum Distance outlier filter is used to compute the unscaled distance (e': euclidean error).
+
+Equations:
+![eq maxdist](images/eq_maxdist.JPG)
+
+Plots:
+![maximumdistance threek](images/maximumdistance_threek.png)
+![maximumdistance multik](images/maximumdistance_multik.png)
+
+#### Generalized ICP (GICP) <a name="gicphead"></a>
+
+This registration algorithm is known to use the Maximum Distance outlier filter on the scaled error (e).
+
+Equations:
+![eq gicp](images/eq_gicp.JPG)
+
+Original reference:
+A.Segal, D.Haehnel, and S.Thrun, "Generalized-ICP", in Robotics: Science and Systems, vol.5, 2009, pp.168-176
+
+### Step Family <a name="stepfamilyhead"></a>
+
+This family of outlier filters is characterized by an influence function that looks like a linear function. Apart from L1, the value of k affects the width of the linear part linking the step. Past the linear behavior, unlike the linear family the influence function becomes equal to -k or k. Usually, inliers are affected by the linear part while the outliers are affected by the -k/k part of the function. The Logistic outlier filter provides a smooth transition between the inliers and outliers parts while Huber provides a sharp transition between the two behaviors. L1 represents the ideal step function therefore it does not have a linear part. The LMICP outlier filter overlaps with the plot of Huber.
+
+![step family plot](images/step_family.png)
+
+#### L1 <a name="l1head"></a>
+
+L1 is a function which is a part of the Lp space family of functions. The L1 outlier filter can't be tuned with a k parameter.
```

Comment:
"The L1 outlier filter is parameter-less, so no parameter needs to be turned."

Reply:
Corrected